### DISCUSSION TITLE
Initialize WebRTC over WebSocket deprecated in v0.23?

### DISCUSSION THREAD

**Discussion by mrlt8** [bluenviron/mediamtx#1845] [Questions] [answered]
Hello,

I've reviewed the release notes and documentation for v0.23, but I couldn't find any explicit information regarding the status of the `/ws` WebSocket endpoint. 

Has Initializing WebRTC over WebSocket been deprecated in favor of WHEP?

Thanks!

**Comment by aler9** [comment, marked as answer]
Hello, yes, the issue with initialization through WebSocket is that it has never been standardized, so it can't be used outside the server. All its features are provided by WHIP/WHEP.

[a comment by an automated account is not shown]
